.github/workflows/python-ci.yml

```diff
@@ -76,17 +80,46 @@ jobs:
           -e AWS_REGION=us-east-1 \
           -e AWS_ACCESS_KEY_ID=dummy \
           -e AWS_SECRET_ACCESS_KEY=dummy \
+          -e POSTGRES_HOST=postgres \
+          -e POSTGRES_PASSWORD=PdwPNS2mDN73Vfbc \
+          -e POSTGRES_DB=polis-test \
           delphi \
           bash -c " \
+            set -e; \
             echo '--- Setting up DynamoDB Tables ---'; \
             python create_dynamodb_tables.py --region us-east-1; \
-            echo '--- Installing Pytest ---'; \
-            pip install pytest; \
+            echo '--- Installing Pytest & Coverage tools ---'; \
+            pip install pytest pytest-cov coverage; \
```

Comment:
fine for now, but https://github.com/compdemocracy/polis/pull/2299 will update the Dockerfile to include dev libs in the `test` build